packages/interface/src/env.js

```diff
@@ -107,6 +108,7 @@ module.exports = createEnv({
     NEXT_PUBLIC_ROUND_LOGO: process.env.NEXT_PUBLIC_ROUND_LOGO,
 
     NEXT_PUBLIC_SEMAPHORE_SUBGRAPH: process.env.NEXT_PUBLIC_SEMAPHORE_SUBGRAPH,
+    NEXT_PUBLIC_TREE_URL: process.env.NEXT_PUBLIC_TREE_URL,
```

Comment:
this is a URL where the tree.json file is uploaded so we can reconstruct the tree, not sure if this should be in the contract.

Reply:
was thinking of a way to simplify configuration, guess probably putting in the contract is also not great in case the source where it's uploaded goes down and cannot be changed. let's keep like this for now